server/server/src/main/java/org/infinispan/server/server/Bootstrap.java

```diff
@@ -0,0 +1,124 @@
+      } catch (IOException e) {
+         stdErr.printf("Could not load logging.properties: %s", e.getMessage());
+         e.printStackTrace(stdErr);
+      }
+
+      for (int i = 0; i < args.length; i++) {
+         String command = args[i];
+         String parameter = null;
+         if (command.startsWith("--")) {
+            int equals = command.indexOf('=');
+            if (equals > 0) {
+               parameter = command.substring(equals + 1);
+               command = command.substring(0, equals);
+            }
+         } else if (command.startsWith("-")) {
+            if (command.length() != 2) {
+               stdErr.println(MSG.invalidShortArgument(command));
+               exitHandler.exit(1);
+            }
+         } else {
+            stdErr.println(MSG.invalidArgument(command));
+            exitHandler.exit(1);
+         }
+         switch (command) {
+            case "-h":
+            case "--help":
+               help(stdOut, true);
+               exitHandler.exit(0);
+               break;
+            case "-c":
+               parameter = args[++i];
+               // Fall through
+            case "--server-config":
+               configurationFile = new File(parameter);
+               break;
+            case "--s":
+               parameter = args[++i];
+               // Fall through
+            case "--server-root":
+               serverRoot = new File(parameter);
+               break;
+            case "-b":
+               parameter = args[++i];
+               // Fall through
+            case "--bind-address":
+               properties.setProperty(Server.INFINISPAN_BIND_ADDRESS, parameter);
+               break;
+            case "-o":
+               parameter = args[++i];
+               // Fall through
+            case "--port-offset":
+               properties.setProperty(Server.INFINISPAN_PORT_OFFSET, parameter);
+               break;
+            case "-v":
+            case "--version":
+               help(stdOut, false);
+               exitHandler.exit(0);
+               break;
+            default:
+               stdErr.println(MSG.unknownParameter(args[i]));
+               exitHandler.exit(1);
+               break;
+         }
+      }
+      try {
+         Runtime.getRuntime().addShutdownHook(new ShutdownHook(exitHandler));
+         Server.log.serverStarting(Version.getBrandName());
+         Server server = new Server(serverRoot, configurationFile, properties);
+         server.setExitHandler(exitHandler);
+         server.run().get();
+      } catch (Exception e) {
+         Server.log.fatal("Error", e);
+      }
+   }
+
+   public static void help(PrintStream stdOut, boolean usage) {
+      stdOut.printf("%s Server %s (%s)\n", Version.getBrandName(), Version.getVersion(), Version.getCodename());
+      if (usage) {
+         stdOut.printf("Usage:\n");
+         stdOut.printf("  -b, --bind-address=<address>  Binds the server to the specified address.\n");
```

Comment:
I am still not sure what port this refers to. Is this hotrod? Admin?

Reply:
Maybe we should put the comments about the descriptions here so they are a bit more descriptive and the README says just to inovke the shell script with -h to get the arguments and information pertaining to them.

Reply:
I would not make the descriptions any more articulate here: this is the server help, not a man page :)